README.md

```diff
@@ -355,13 +355,51 @@ require("zk.ui").get_pick_notes_list_api_selection(options)
 ```
 
 ## Example Mappings
+
+To add key mappings only for Markdown documents, you can use `ftplugin`. First, make sure it is enabled in your Neovim config:
+
+```viml
+filetype plugin on
+```
+
+Then, create a new file under `~/.config/nvim/ftplugin/markdown.lua` to setup your Markdown key mappings.
```

Comment:
This will enable the key bindings for all Markdown buffers, even if they are not in a notebook. Maybe there's a way to enable this only for `zk` notes? Adding a Lua API to determine if a path is part of a notebook?

Reply:
`require("zk.util").notebook_root(path) ~= nil`

Reply:
Thanks, it worked. I updated the README